dips/dip-158.md

```diff
@@ -0,0 +1,805 @@
+---
+dip: 158
+title: P2M (eCommerce) Payments Off-Chain Protocol
+authors: Yinon Zohar, Daniel Prinz
+status: Draft
+type: Standard
+created: 03/15/2021
+---
+
+# Summary
+This DIP is an extension of the Off-Chain protocol for enabling eCommerce P2M payments (Peer To Merchant) of the following types:
```

Comment:
This is probably where we should at least say DIP X, where 0 < X < 2 and say that it leverages the off-chain protocol defined therein. And also uses the extension for the return result defined in DIP-161: https://dip.diem.com/dip-161/

Reply:
Added reference to DIP-1 with link.